test/hotspot/jtreg/vmTestbase/nsk/jvmti/ResourceExhausted/resexhausted001/TestDescription.java

```diff
@@ -37,7 +38,7 @@
  *
  * @library /vmTestbase
  *          /test/lib
- * @run main/othervm/native/timeout=240
+ * @run main/othervm/native/manual/timeout=240
```

Comment:
What is the reason to make this test manual?

Reply:
These tests are unreliable.

Reply:
Sorry, I've missed the answer to the same question.

Reply:
Thanks Leonid, I didn't really have a better thing to do with these tests because git made me take them off the problem list where they've been on and off for years.